lua/entities/gmod_wire_clutch.lua

```diff
@@ -274,9 +274,9 @@ function ENT:ApplyDupeInfo(ply, ent, info, GetEntByID)
 		Ent2 = GetEntByID(v.Ent2, game.GetWorld())
 
 		if IsValid(Ent1) and
-		   Ent1 ~= Ent2 and
```

Comment:
Looks like it missed some spaces here

Reply:
Corrected in the last commit. Could not point you as a co-author for some reason.